common/djangoapps/student/views/dashboard.py

```diff
@@ -530,6 +530,10 @@ def student_dashboard(request):  # lint-amnesty, pylint: disable=too-many-statem
     empty_dashboard_message = configuration_helpers.get_value(
         'EMPTY_DASHBOARD_MESSAGE', None
     )
+    disable_unenrollment = configuration_helpers.get_value(
+        'DISABLE_UNENROLLMENT',
```

Comment:
It looks like this code allows site configuration to override the global config, but this other location will only look at the global config:

https://github.com/openedx/edx-platform/pull/29326/files#diff-be6de6b401b7cfa4c2c9818304e64da16dd60d78aa1274e9956fc946aa62fbf0R406

(Am I just missing something?)

Reply:
Thanks for catching this, @timmc-edx. I've added the override and tested that it works.